austation/code/modules/mining/necropolis_chests.dm

```diff
@@ -0,0 +1,114 @@
+/obj/structure/closet/crate/necropolis/tendril/PopulateContents()
+  var/loot = rand(1,29)
+  switch(loot)
+    if(1)
+      new /obj/item/shared_storage/red(src)
+    if(2)
+      new /obj/item/clothing/suit/space/hardsuit/cult(src)
+    if(3)
+      new /obj/item/soulstone/anybody(src)
+    if(4)
+      new /obj/item/katana/cursed(src)
+    if(5)
+      new /obj/item/clothing/glasses/godeye(src)
+    if(6)
+      new /obj/item/reagent_containers/glass/bottle/potion/flight(src)
+    if(7)
+      new /obj/item/pickaxe/diamond(src)
+    if(8)
+      new /obj/item/rod_of_asclepius(src)
+    if(9)
+      new /obj/item/organ/heart/cursed/wizard(src)
+    if(10)
+      new /obj/item/ship_in_a_bottle(src)
+    if(11)
+      new /obj/item/clothing/suit/space/hardsuit/ert/paranormal/beserker(src)
+    if(12)
+      new /obj/item/jacobs_ladder(src)
+    if(13)
+      new /obj/item/nullrod/scythe/talking(src)
+    if(14)
+      new /obj/item/nullrod/armblade(src)
+    if(15)
+      new /obj/item/guardiancreator/hive(src)
+    if(16)
+      new /obj/item/warp_cube/red(src)
+    if(17)
+      new /obj/item/wisp_lantern(src)
+    if(18)
+      new /obj/item/immortality_talisman(src)
+    if(19)
+      new /obj/item/gun/magic/hook(src)
+    if(20)
+      new /obj/item/voodoo(src)
+    if(21)
+      new /obj/item/grenade/clusterbuster/inferno(src)
+    if(22)
+      new /obj/item/reagent_containers/food/drinks/bottle/holywater/hell(src)
+      new /obj/item/clothing/suit/space/hardsuit/ert/paranormal/inquisitor(src)
+    if(23)
+      new /obj/item/book/granter/spell/summonitem(src)
+    if(24)
+      new /obj/item/book_of_babel(src)
+    if(25)
+      new /obj/item/borg/upgrade/modkit/lifesteal(src)
+      new /obj/item/bedsheet/cult(src)
+    if(26)
+      new /obj/item/clothing/neck/necklace/memento_mori(src)
+    if(27)
+      new /obj/item/reagent_containers/glass/waterbottle/relic(src)
+    if(28)
+      new /obj/item/reagent_containers/glass/bottle/necropolis_seed(src)
+    if(29)
+      new /obj/item/tank/internals/occult(src)

```

Comment:
ok this is cool, but if bee adds a new tendril item, this needs to all be updated.

Reply:
I'll see if I can make some scuffed way to make it not just override it